### PR TITLE
Create feedstock to xeus-sqlite

### PR DESCRIPTION
**Pull request by marimeireles**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/xeus-sqlite/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  run:
+    # Even though cppzmq, xtl and nlohmann_json are header-only,
+    # they are used at runtime by the c++ interpreter.
+    - {{ pin_compatible('cppzmq', max_pin='x.x') }}
+    - {{ pin_compatible('xtl', max_pin='x.x') }}
+    - {{ pin_compatible('nlohmann_json', max_pin='x.x') }}
+
+test:
+  requires:
+    - jupyter_kernel_test
```
Comment (marcelotrevisani) [inline recipes/xeus-sqlite/meta.yaml:39]:
```suggestion
```

Comment (marcelotrevisani) [reply recipes/xeus-sqlite/meta.yaml:39]: as you are just testing if those binaries are present, you do not need these lines

File: recipes/xeus-sqlite/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
+{% set name = "xeus-sqlite" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```
Comment (marcelotrevisani) [inline recipes/xeus-sqlite/meta.yaml:9]:
```suggestion
```

File: recipes/xeus-sqlite/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  url: https://github.com/jupyter-xeus/xeus-sqlite/archive/{{ version }}.tar.gz
+  sha256: eacc8591fd26c009a407d88867796880b13ea27113931157e4abe52324ba4b74
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
```
Comment (marcelotrevisani) [inline recipes/xeus-sqlite/meta.yaml:19]:
```suggestion
    - cmake
    - make  # [unix]
    - nmake  # [win]
```

Comment (marimeireles) [reply recipes/xeus-sqlite/meta.yaml:19]: Thanks for the review @marcelotrevisani :)

File: recipes/xeus-sqlite/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    - make # [unix]
+    - nmake # [win]
+  host:
+
```
Comment (marcelotrevisani) [inline recipes/xeus-sqlite/meta.yaml:22]:
```suggestion
```

**Pull request merged**